src/com/boxboat/jenkins/pipeline/deploy/BoxDeploy.groovy

```diff
@@ -215,6 +215,38 @@ class BoxDeploy extends BoxBase<DeployConfig> implements Serializable {
         }
     }
 
+    Map<Object, Object> getImageTagsMetadata() {
+        def buildVersions = Config.getBuildVersions()
+        Map<Object, Object> imageTagsMetadata = new LinkedHashMap<>()
+        config.images.each { Image image ->
+            def event = deployment.event
+            def imageOverridesCl = { Image imageOverride ->
+                if (imageOverride.path == image.path) {
+                    event = imageOverride.event
+                }
+            }
+            config.imageOverrides.each imageOverridesCl
+            deployment.imageOverrides.each imageOverridesCl
+            String version = ""
+            Map<Object, Object> imageMetaData
+            if (Utils.isImageTagEvent(event)) {
+                image.tag = Utils.imageTagFromEvent(event)
+                imageMetaData = buildVersions.getEventImageVersionMetadata(image.tag, image)
```

Comment:
@jordanjennings the method you referred to is being used here in `getImageTagsMetadata`